src/components/visualizations/chart/highcharts/plugins/adjustTickAmount.ts

```diff
@@ -44,8 +45,8 @@ function isUserSetExtremesOnAnyAxis(chart: Highcharts.Chart): boolean {
  *      1: move zero index to right
  */
 function getDirection(primaryAxis: IHighchartsAxisExtend, secondaryAxis: IHighchartsAxisExtend): number {
-    const { tickPositions: primaryTickPosition } = primaryAxis;
-    const { tickPositions: secondaryTickPosition } = secondaryAxis;
+    const { tickPositions: primaryTickPosition = [] } = primaryAxis;
```

Comment:
ES6 destructuring default values only kick in if the value is undefined. null, false and 0 are all still values!
https://wesbos.com/destructuring-default-values/

Is there a chance that `primaryTickPosition`, `secondaryTickPosition` be `null`, `false` or `0`?

Reply:
I have updated.
With latest impl, we validate axes in caller function so we don't need it anymore.